### PR TITLE
43 draw deployment diagram

### PR DESCRIPTION
**Pull request by IsabelleTornqvist**
Drawing of deployment diagram, including mock data.

### CODE REVIEW

**Review by timjon — DISMISSED** [review]
Version numbers

File: Application/src/view/visuals/Draw.java [hunk 1]
```diff
@@ -117,11 +128,34 @@ public void addClass(String name) {
...
      */
-
     public void addClassDiagramClass (String name){
         allClassDiagramClasses.add(new ClassDiagramClass(name));
     }
 
+    /**
+     * Adds nodes to arraylist, to be drawn
+     * @param device
+     *
```
Comment (pontuslaestadius) [inline Application/src/view/visuals/Draw.java]: Remove empty comment line

File: Application/src/view/visuals/Draw.java [hunk 2]
```diff
@@ -304,9 +353,29 @@ private void renderClass() {
...
+    }
+
+    /**
+     *Matrix structure, used for class and deployment diagrams
+     * @param diagram
+     * @param columns
+     * @param space
+     * @param canvas
+     */
+    private void Matrix(ArrayList<Renderable> diagram, int columns, int space, Canvas canvas){
```
Comment (pontuslaestadius) [inline Application/src/view/visuals/Draw.java]: Uppercase method naming

**Review by pontuslaestadius — CHANGES_REQUESTED** [review]
Also add text for the @param tags

**Review by pontuslaestadius — APPROVED** [review]

**Review by kosara-g — APPROVED** [review]
Go go go

**Pull request merged**